components/SupportUs.js

```diff
@@ -1,99 +1,74 @@
+import React from 'react';
+
+export const supportItems = [
+  {
+    href: 'https://slack.com/media-kit',
+    target: '_blank',
+    className: 'block relative text-center w-2/3 px-4 py-4 sm:p-0 sm:w-1/3 md:w-1/3 lg:w-1/5',
+    rel: 'noopener noreferrer',
+    src: '/img/supportus/slack.webp',
+    title: 'Slack - Free Standard Subscription.'
+  },
+  {
+    href: 'https://toast.ninja/',
+    target: '_blank',
+    className: 'block relative text-center w-2/3 px-4 py-4 sm:p-0 sm:w-1/3 md:w-1/3 lg:w-1/5',
+    rel: 'noopener noreferrer',
+    src: '/img/supportus/toast.webp',
+    title: 'Toast - Free services.'
+  },
+  {
+    href: 'https://www.netlify.com/',
+    target: '_blank',
+    className: 'block relative text-center w-2/3 px-4 py-4 sm:p-0 sm:w-1/3 md:w-1/3 lg:w-1/5',
```

Comment:
`className` should not be the part of this array. Since, these are CSS properties, add the classes in `<a>` tag only.

Reply:
I tried doing that , however it doesn't give the expected result